### ISSUE TITLE
Advanced Loss Functions

### ISSUE THREAD

**Issue by michalharakal** [sk-ai-net/skainet#302]
-  BinaryCrossEntropy 	
- CategoricalCrossEntropy 	
- CrossEntropy
- Hinge
- Huber
- MAE
- Poisson
- LogCosh
